pipeline_steps/run_tests.groovy

```diff
@@ -0,0 +1,72 @@
+def run_irr_tests() {
```

Comment:
Last thing, can you call this irr_tests.groovy or something similar, so that the file name reflects the functions defined within.